frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -0,0 +1,252 @@
+        <div className="grid-col-4 text-base">Value type</div>
+        <div className="grid-col-auto">{item.valueType}</div>
+      </div>
+      <div className="grid-row margin-bottom-05 border-base-lighter border-top-1px padding-top-1">
+        <div className="grid-col-4 text-base">
+          {item.acceptedFormat && "Accepted format"}
+          {item.acceptedValues && "Accepted value(s)"}
+          {item.acceptedExample && "Example(s)"}
+        </div>
+        <ul className="grid-col-8 prime-ul">
+          {item.values?.map((value, valueIndex) => (
+            <li
+              key={`${item.colHeader}-value-${valueIndex}`}
+              dangerouslySetInnerHTML={{ __html: `${value}` }}
+            />
+          ))}
+        </ul>
+      </div>
+    </div>
+  );
+};
+
+/* eslint-disable jsx-a11y/anchor-has-content */
+const CsvSchemaDocumentation = () => {
+  useDocumentTitle("CSV Schema Documentation");
+
+  return (
+    <div className="prime-container prime-container-padding-2x card-container">
+      <div className="tablet:grid-col-8 usa-prose">
+        <section id="anchor-top">
+          <span className="text-base text-italic">Updated: May 9, 2022</span>
+          <h2 className="margin-top-0 ">
+            CSV template guide{" "}
+            <span className="text-secondary bg-white border-1px border-secondary font-body-3xs padding-x-1 padding-y-05 text-secondary margin-left-2 text-ttbottom">
+              Pilot program{" "}
+            </span>
+          </h2>
+          <p className="usa-intro text-base">
+            How to format a CSV spreadsheet for submission to public health
+            departments through SimpleReport.
+          </p>
+          <p>
+            <strong>In this guide</strong>
+          </p>
+          <ul>
+            <li>
+              <a href="#formatting-guidelines" className="usa-link">
+                General formatting guidelines
+              </a>
+            </li>
+            {schema.fields.map((field, fieldIndex) => {
+              return (
+                <div key={`toc-${fieldIndex}`} className="">
+                  {field.sections?.map((section, sectionIndex) => {
+                    return (
+                      <li key={`toc-${fieldIndex}-${sectionIndex}`}>
+                        <a href={`#${section.slug}`} className="usa-link">
+                          {section.title}
+                        </a>
+                      </li>
+                    );
+                  })}
+                </div>
+              );
+            })}
+          </ul>
+          <p>
+            <strong>Resources</strong>
+          </p>
+          <ul>
+            <li>
+              <a
+                href="/getting-started/testing-facilities/csv-schema"
+                className="usa-link"
+              >
+                Standard schema with example data
+              </a>
+            </li>
+          </ul>
+        </section>
+
```

Comment:
```suggestion
 <section className="border-top-1px border-ink margin-top-9">
          <h3 id="formatting-guidelines" className="font-body-lg margin-y-1">
          Preparing and uploading a spreadsheet
          </h3>
          <ol className="usa-process-list">
            <li className="usa-process-list__item">
              <h4 className="usa-process-list__heading">
                Create or export your spreadsheet
              </h4>
              <p className="margin-top-05">
               If your organization already uses a set spreadsheet format for results, you may have to adjust it to match the SimpleReport template. If you don’t have one, use the {" "}
                <a
                  href="https://SimpleReport.cdc.gov/assets/csv/SimpleReport-StandardCSV-ExampleData-20220509.csv"
                  className="usa-link"
                >
                  spreadsheet template file
                </a>{" "} as a starting point. 
              </p>
            </li>
            <li className="usa-process-list__item">
              <h4 className="usa-process-list__heading">
                Format using the SimpleReport template
              </h4>
              <p>
                In your spreadsheet, include all column headers in the spreadsheet template and guidelines below, with no extras. Copy column header names exactly. Be sure to include every column in the template, even if you don’t have data for every field. 
              </p>
            </li>
            <li className="usa-process-list__item">
              <h4 className="usa-process-list__heading">
                Enter your data</h4>
              <p>
               Following the spreadsheet guidelines below, enter properly formatted values in the relevant fields. Some fields require data, while others don’t.
              </p>
            </li>
            <li className="usa-process-list__item">
              <h4 className="usa-process-list__heading">Export or save your data</h4>
              <p>Make sure your spreadsheet is in a CSV format. SimpleReport does’t accept .XLS, .XLXS, or other formats.</p>
            </li>
            <li className="usa-process-list__item">
              <h4 className="usa-process-list__heading">Use the uploader on SimpleReport</h4>
              <p>Visit the “[Upload spreadsheet](url)” tab under “Results” in the main SimpleReport navigation. Select your CSV by dragging the file from a folder to the upload area, or browse your computer to find and open it. Once you click Upload your CSV, SimpleReport will check your file to see if it matches the template. If it accepts the file, you’ll see a confirmation message.</p>
              <h4 className="usa-process-list__heading">Fix any errors</h4>
              <p>If SimpleReport finds any errors in the spreadsheet formatting or data, it will recommend how to fix them. Once you’ve made the recommended changes in your spreadsheet, save it, and then upload it again. 
</p>
            </li>
          </ol>
        </section>
```
Note: will need to add proper link on step 5, "Upload spreadsheet"

Reply:
This update simplifies the upload guide and adds it to the schema documentation. We won't need the documentation previously known as "CSV upload guide," but should rename this page "Spreadsheet upload guide."

cc @nathancrtr @vz3

Reply:
Love it, I've been thinking that the "upload guide" was a little redundant 👍